crates/wasm-pkg-common/src/config.rs

```diff
@@ -23,15 +21,26 @@ pub struct Config {
     default_registry: Option<Registry>,
     namespace_registries: HashMap<Label, Registry>,
     package_registry_overrides: HashMap<PackageRef, Registry>,
+    // Note: these are only used for hard-coded defaults currently
+    fallback_namespace_registries: HashMap<Label, Registry>,
     registry_configs: HashMap<Registry, RegistryConfig>,
 }
 
 impl Default for Config {
     fn default() -> Self {
+        let fallback_namespace_registries = [
```

Comment:
Nit: It might be good to have these as a const, even if we don't publicly export them (which it might be valuable to do)

Reply:
Done.